### DISCUSSION TITLE
Connection between two hubs

### DISCUSSION THREAD

**Discussion by Gcopper1984** [pybricks/support#503] [Ideas]
Good morning to everyone and thank you for your great work.

I have a question: is there any way to make a connection between two hubs and send data between them?
I saw this topic on the forum:

"Dual Technic Hub Connection for passing data and control #280"

but I do not understand if it is already implemented or not.

Thank you

**Comment by laurensvalk** [comment]
> but I do not understand if it is already implemented or not.

It's not implemented yet. But some progress is being made ;)

**Reply by Gcopper1984** [reply]
Great!
